package.json

```diff
@@ -1,8 +1,8 @@
 {
   "private": true,
   "workspaces": [
-    "packages/*",
-    "apps/*"
+    "apps/*",
+    "packages/*"
```

Comment:
:bulb: Task filtering seemed to fail if the order of these workspaces wasn't reversed. :shrug: